docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -83,19 +80,21 @@ Use this to orient the ADC to determine the location to insert your wires.
 {{% /alert %}}
 
 Insert the MCP3008 into your breadboard so that it bridges both sides of the divide.
-Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your PI to connect the pins with wires as follows:
+Then, use the rows on the side of your MCP3008's pins and the GPIO pins on your Pi to connect the pins with wires as follows:
 
 <!-- prettier-ignore -->
 | MCP3008 ADC Pin | Raspberry Pi Pin |
 | ----------- | ---------------- |
-| VDD | 3.3V |
-| VREF | 3.3V |
+| VDD | 5V |
+| VREF | 5V |
 | AGND | GND |
 | DGND | GND |
 | CLK | SCLK |
 | DOUT | MISO |
 | DIN | MOSI |
-| CS/SHDN | GPIO8 |
+| CS/SHDN | 24GPIO8 |
+
+The 5V Raspberry Pi pins go into the power rail of the breadboard (the red plus sign column). This will be used to share the 5V output from the Raspberry Pi Pin 4 which is 5V power. Thus one additional wiring from the top of the power rail of the breadboard to the Raspberry Pi [Pin 4 (5V)](https://pinout.xyz/pinout/5v_power) is necessary.
```

Comment:
I think we can take out some extra words here. How about this:
```suggestion
Use an additional wire to wire Raspberry Pi [pin 4 (a 5 volt power pin)](https://pinout.xyz/pinout/5v_power) to the power rail of the breadboard (the red plus sign column).
This brings the 5V power output from the Raspberry Pi to the ADC and the sensor.
```
Also put one sentence per line ;)